src/_includes/docs/swift-package-manager/how-to-enable-disable.md

```diff
@@ -25,9 +25,11 @@ Using the Flutter CLI to run an app [migrates the project][addSPM] to add
 Swift Package Manager integration.
 This makes your project download the Swift packages that
 your Flutter plugins depend on.
-If you disable the Swift Package Manager feature,
-you will need to [remove Swift Package Manager integration][removeSPM] from apps
-that are migrated.
```

Comment:
The previous statement was incorrect. You had to undo the integration if you switched to the stable branch (which also implicitly disables the SPM feature). If you disabled the SPM feature but stayed on the master channel, the app would continue to build fine. I had misunderstood how this worked!